Brokerages/Alpaca/AlpacaApiBase.cs

```diff
@@ -723,9 +729,10 @@ public IEnumerable<TradeBar> DownloadTradeBars(Symbol symbol, DateTime startTime
                 }
                 catch (Exception e)
                 {
+                    Log.Error(e);
                     if (e.InnerException != null)
                         OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Warning, 100, e.InnerException.Message));
-                    throw;
+                    throw e;
```

Comment:
I believe when re-throwing exceptions we should always use `throw;` instead of `throw e;`, since the latter resets the call stack to the current location and the original stack trace would be lost.

Reply:
Oh I see.